Game/src/Controller.java

```diff
@@ -0,0 +1,43 @@
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.Scene;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class Controller implements Initializable {
+
+    public static Scene prevScene;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+    }
+
+    @FXML
```

Comment:
In my opinion we should stay away from FXML as it is largely incompatible with some of the more complex things we plan to add to the game later on. Asking @hikari1734  for her opinion

Reply:
I agree with @smrflynn on this one.